src/lib/services/transfer/service/endpoint-manager/task.ts

```diff
@@ -236,7 +236,7 @@ export const getPauseInfo = function (
   task_id,
   options?,
   sdkOptions?,
-): Promise<JSONFetchResponse<Globus.Transfer.AdminPauseDocumentResponse>> {
+): Promise<JSONFetchResponse<Globus.Transfer.PauseRuleLimitedDocument[]>> {
```

Comment:
Is this an array? 

https://docs.globus.org/api/transfer/advanced_collection_management/#get_task_pause_info is showing:

```json
{
    "DATA_TYPE": "pause_info_limited",
    "pause_rules": [... list of pause_rule_limited documents...],
    "source_pause_message": null,
    "destination_pause_message": "Disk problems, pausing all tasks until we resolve",
    "source_pause_message_share": null,
    "destination_pause_message_share": null
}
```

The [internal type](https://github.com/globusonline/webapp/blob/master/packages/%40globus/types/transfer/index.d.ts#L401C1-L408C4) is:

```ts
  interface PauseInfoLimitedDocument extends TransferEntity {
    DATA_TYPE: `${DATA_TYPE.pause_info_limited}`;
    pause_rules: Array<PauseRuleLimitedDocument>;
    source_pause_message: string | null;
    destination_pause_message: string | null;
    source_pause_message_share: string | null;
    destination_pause_message_share: string | null;
  }
```

```suggestion
): Promise<JSONFetchResponse<Globus.Transfer.PauseRuleLimitedDocument>> {
```